static/app/views/issueList/overview.tsx

```diff
@@ -170,6 +171,7 @@ function IssueListOverview({router}: Props) {
   const [queryMaxCount, setQueryMaxCount] = useState(0);
   const [error, setError] = useState<string | null>(null);
   const [issuesLoading, setIssuesLoading] = useState(true);
+  const [issuesSuccessfullyLoaded, setIssuesSuccessfullyLoaded] = useState(false);
```

Comment:
I needed to know whether a fetch had been performed or not, which was not possible with the existing state. This should be able to be replaced with `isSuccess` if we ever switch to react query.